browser/ui/brave_browser.cc

```diff
@@ -200,6 +204,35 @@ void BraveBrowser::TabCustomTitleChanged(
   }
 }
 
+content::WebContents* BraveBrowser::AddNewContents(
+    content::WebContents* source,
+    std::unique_ptr<content::WebContents> new_contents,
+    const GURL& target_url,
+    WindowOpenDisposition disposition,
+    const blink::mojom::WindowFeatures& window_features,
+    bool user_gesture,
+    bool* was_blocked) {
+  // Only handle NEW_FOREGROUND_TAB disposition (target="_blank" links)
```

Comment:
One of the things that `target=_blank` does is automatically break the `window.opener` link, basically doing the same as [`rel=noopener`](https://developer.mozilla.org/en-US/docs/Web/HTML/Reference/Attributes/rel/noopener). Did you check the right pane navigation gets a null `window.opener`?

Reply:
In `split_view::MaybeRedirectToRightPane()`,  `NavigationParams` is created and default `NavigationParams::opener` is used. It's `nullptr` by default.

Reply:
Addressed by[Refactor Refactor split view window.open() redirect](https://github.com/brave/brave-core/pull/32608/commits/72e63f8c4bc8dd39670b117fd7a14b38ccd7e0f3)
Didn't set opener when routed from window.open.